src/order.rs

```diff
@@ -0,0 +1,46 @@
+use async_graphql::{Enum, SimpleObject};
+use bson::Uuid;
+use serde::{Deserialize, Serialize};
+
+use crate::{http_event_service::OrderEventData, invoice::Invoice};
+
+/// Foreign type of an order.
+#[derive(Debug, Serialize, Deserialize, SimpleObject, Clone)]
+#[graphql(unresolvable)]
+pub struct Order {
+    /// UUID of the order.
+    pub _id: Uuid,
+    /// UUID of user connected with Order.
+    #[graphql(skip)]
+    pub user_id: Uuid,
```

Comment:
do you need this if no auth is required for the entity resolver?

Reply:
No, i can remove it now. Also the authentication module